design/oss_suite_release.md

```diff
@@ -230,14 +229,16 @@ will include:
     links to artifacts for each component version included in the release
   - Note: when possible, links to artifacts should link directly to the canonical
     source, rather than linking to the component README (eg link directly to Dockerhub, etc)
+  - Bumping the version of Conjur pinned in the [Conjur helm chart](https://github.com/cyberark/conjur-oss-helm-chart)
```

Comment:
I thought we decided this was out of scope.  Why the change to bring it in?

Reply:
we don't list this as out of scope in the doc, and I don't expect much effort needed to submit a PR to the helm chart repo with a conjur version bump. I think this is doable, and I'm fine including it based on my current understanding of it